base_ubl_payment/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+================
+Base UBL Payment
+================
+
+This module is a small module above the *base_ubl* module;
+it adds the generation of the *PaymentMeans* UBL block.
+I decided to make it a separate module because it depends
+on the module *account_payment_unece* which itself depend on
+*account_banking_payment_export*, and I didn't want to add
+these dependencies on the *base_ubl* module.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/226/11.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/edi/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smash it by providing detailed and welcomed feedback.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Alexis de Lattre <alexis.delattre@akretion.com>
+
```

Comment:
You must add you as a contributor.

Reply:
@cubells I already did it in the  CONTRIBUTORS.rst file, so the readme will be automatically fixed. Is it ok with that? This way I will save another commit and push :smile: